### DISCUSSION TITLE
by using pydpf post how do i extract the FSI results?

### DISCUSSION THREAD

**Discussion by velpulagopiraju** [ansys/pydpf-post#573] [Q&A]
is it possible to extract the FSI results by using the pydpf.if possible can you please suggest me the way how to proceed ..

**Comment by PProfizi** [comment]
Hello @velpulagopiraju,
To know if PyDPF can extract your FSI results, could you please indicate the result files formats you want to read.
To know which Ansys file formats PyDPF can open, please refer to the main landing page of any PyDPF library such as [here](https://post.docs.pyansys.com/version/stable/).

**Reply by velpulagopiraju** [reply]
i do have .cas&.dat result files related to the fluid simulation and .rst result files related to the mechanical simulation.
